q2_fmt/tests/test_engraftment.py

```diff
@@ -631,6 +629,7 @@ def test_mann_whitney_pairwise_against_each(self):
                                     compare='all-pairwise',
                                     p_val_approx='asymptotic')
 
+        print(stats_data)
```

Comment:
Some git link and a print. You can use `git checkout REF -- some/file/here` to pull an older/different version from somewhere else (like main/master).

Reply:
Thanks for catching that print @ebolyen! The git link is part of a comment above the mann_whitney tests that shows how the expected data was manually calculated - let me know if you want that to be removed!

Reply:
ah, typo on my end, "git lint" was what I meant to say. The test data was shuffled in a way that doesn't matter to this feature, creating some "lint", but it does look better the way you re-arranged it, so I'm happy with it as is.

Reply:
ahh, gotcha! that makes more sense re: your comment above. i'll keep that in mind for next time, but leave as-is for this PR. thank you! 🙂